app/views/places/show.html.erb

```diff
@@ -0,0 +1,28 @@
+<h1> <%= @places.name %> - places details</h1>
```

Comment:
In show action we should refer to a single record, so the plural variable name may be misleading. Moreover, it seems that it's empty as you don't have it defined in the controller

Reply:
OK